src/gui/mzroll/main.cpp

```diff
@@ -55,8 +55,9 @@ int main(int argc, char *argv[])
 {
 
     QApplication app(argc, argv);
-    qApp->setOrganizationName("ElucidataInc");
-    qApp->setApplicationName("El-Maven");
+    qApp->setOrganizationName("Elucidata");
+    qApp->setApplicationName("El-MAVEN");
+    qApp->setApplicationVersion("v0.4");
```

Comment:
@sahil21 Do we have to set this manually before every release?

Reply:
@shubhra-agrawal Added a commit to resolve this.